phantom_token.c

```diff
@@ -811,3 +808,70 @@ static char* set_client_credential_configuration_slot(ngx_conf_t *config_setting
 
     return "invalid_client_credential";
 }
+
+/*
+ * Add the error response as a JSON object that is easier to handle than the default HTML response that NGINX returns
+ * Note that for SPAs, the OAuth Proxy plugin runs before this one and adds CORS headers so that the browser can read this payload
```

Comment:
This module can be used independently of the OAuth Proxy plug-in. As a result, the reader of this comment might not even be aware of what this other plug-in is. Seems like more context is needed.

Reply:
Some copying and pasting here - I removed that comment. People using phantom token module or reading its code should not need to think about web concerns. Instead they are dealt with by also adding the oauth proxy module. This is explained in website SPA tutorials.